Aztec/Classes/Formatters/TextListFormatter.swift

```diff
@@ -37,6 +37,39 @@ class TextListFormatter
         let listParagraphs = string.paragraphRanges(spanningRange: listRange)
         return applyList(ofStyle: style, toString: string, atRanges: listParagraphs)
     }
+
+    func refreshList(inString string: NSMutableAttributedString, atRange range: NSRange) -> NSRange? {
```

Comment:
I think this method requires some documentation explaining:

- Why does this method need to refresh the list?
- How does the refresh work?